packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -2492,7 +2596,7 @@ namespace Ifpack2 {
 
       inline
       void 
-      serialGemv(const local_ordinal_type &blocksize,
+      SerialGemv(const local_ordinal_type &blocksize,
```

Comment:
In future revisions, we should factor out these BLAS functions into kokkos-kernels.

Reply:
Yes sometime but not right now. I put this another implementation separately from kokkoskernels because of different performance characteristics. This functions are used to block sparse matrix vector multiplication where input matrix is packed and output matrix is tpetra multivector format. I tested a few variations and what I found is to use atomic add and parallelize over columns in a row. Also, for performance reason, here we specialize blocksize with a template parameter and I give such information explicitly. With kokkos interface, it is ideal to use view interface. Then, templatized blocksize porbably won't be effective.

Reply:
@kyungjoo-kim I see, these are specialized for the block sparse mat-vec use case.  It might be better to move them into a separate file so they can be tested separately, but it sounds like they wouldn't be more generally useful outside of the block sparse mat-vec case.